From 0a7d2bc31fe2249802e6c6f001911f68f2c91d67 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Fri, 17 Apr 2020 15:31:16 +0300
Subject: [PATCH] [SW-10446] fit: add preboot configuration

It will be used to point the image to be loaded by preboot e.g. uboot
---

diff --git a/board/habanalabs/common/uboot.its b/board/habanalabs/common/uboot.its
index a2811c7..22406be 100644
--- a/board/habanalabs/common/uboot.its
+++ b/board/habanalabs/common/uboot.its
@@ -105,5 +105,10 @@
 			loadables = "btl-s1", "btl-s2", "btl-s3", "btl-s4",
 				    "img-ptr", "uboot-env", "preboot";
 		};
+
+		preboot-conf {
+			description = "Habanalabs device preboot";
+			loadables = "uboot";
+		};
 	};
 };
